db/data/syllabusesModel.js

```diff
@@ -18,4 +21,7 @@ db.once('open', () => {
 
 const SyllabusModel = mongoose.model('syllabuses', syllabusesSchema);
 
-module.exports = SyllabusModel;
+module.exports = {
+  SyllabusModel,

```

Comment:
I'm guessing this is exported to your **db/index.js**? And then the **db/index.js** will be exported to your **server/index.js** to use for CRUD ops?

Reply:
Exactly! This is also legacy code other than the fact that I needed to also export the `connect()` function, so I had re-write this to export two things instead of just one.